core-plugins/src/main/java/io/cdap/plugin/batch/aggregator/function/NumberSelection.java

```diff
@@ -67,6 +73,10 @@ public StructuredRecord select(StructuredRecord record1, StructuredRecord record
         return compareFloat((Float) val1, (Float) val2) > 0 ? record1 : record2;
       case DOUBLE:
         return compareDouble((Double) val1, (Double) val2) > 0 ? record1 : record2;
+      case BYTES:
+        return compareBytes((ByteBuffer) val1, (ByteBuffer) val2) > 0 ? record1 : record2;
```

Comment:
I think it can also be `byte[]`, see `io.cdap.cdap.api.data.format.StructuredRecord#getDecimal`

Reply:
got it